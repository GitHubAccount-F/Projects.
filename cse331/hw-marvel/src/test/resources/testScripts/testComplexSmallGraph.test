#Tests the FindPath on a small complex graph

#Creates graph
LoadGraph graph1 complexSmallGraph.csv

#Find's the path between characters
FindPath graph1 SpiderMan GhostRider

FindPath graph1 Galactus Hulk

FindPath graph1 Thor Venom

FindPath graph1 BlackPanther Loki

FindPath graph1 Storm Storm