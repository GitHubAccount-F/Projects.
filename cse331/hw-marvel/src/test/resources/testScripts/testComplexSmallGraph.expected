#Tests the FindPath on a small complex graph

#Creates graph
loaded graph graph1

#Find's the path between characters
path from SpiderMan to GhostRider:
SpiderMan to CaptainAmerica via Civil-War
CaptainAmerica to Hulk via Ragnarok
Hulk to Magneto via X-Men
Magneto to GhostRider via Apocalypse

path from Galactus to Hulk:
no path found

path from Thor to Venom:
Thor to CaptainAmerica via Ragnarok
CaptainAmerica to SpiderMan via Civil-War
SpiderMan to Venom via Spider-Verse

path from BlackPanther to Loki:
BlackPanther to CaptainAmerica via Civil-War
CaptainAmerica to Loki via Ragnarok

path from Storm to Storm: